Formula/kyuubi.rb

```diff
@@ -0,0 +1,52 @@
+class Kyuubi < Formula
+  desc "Serverless SQL on data warehouses and lakehouses"
+  homepage "https://kyuubi.apache.org/"
+  url "https://www.apache.org/dyn/closer.lua?path=kyuubi/kyuubi-1.6.1-incubating/apache-kyuubi-1.6.1-incubating-bin.tgz"
+  mirror "https://archive.apache.org/dist/kyuubi/kyuubi-1.6.1-incubating/apache-kyuubi-1.6.1-incubating-bin.tgz"
+  version "1.6.1-incubating"
+  sha256 "a64a6e5d71b39b0e494597282493250df1c74714fc2bba5057946926c0a32ba2"
+  license "Apache-2.0"
+  head "https://github.com/apache/kyuubi.git", branch: "master"
+
+  def create_bin_with_envs(target = "kyuubi", source = "kyuubi")
+    (bin/target).write_env_script libexec/"bin"/source,
+      JAVA_HOME:            Language::Java.overridable_java_home_env("1.8")[:JAVA_HOME],
+      SPARK_HOME:           "${SPARK_HOME:-#{Formula["apache-spark"].opt_libexec}}",
+      KYUUBI_HOME:          libexec.to_s,
+      KYUUBI_LOG_DIR:       "${KYUUBI_LOG_DIR:-#{var}/kyuubi/logs}",
+      KYUUBI_PID_DIR:       "${KYUUBI_PID_DIR:-#{var}/kyuubi/pid}",
+      KYUUBI_WORK_DIR_ROOT: "${KYUUBI_WORK_DIR_ROOT:-#{var}/kyuubi/work}"
+  end
+
```

Comment:
If it depends on Java and apache-spark you should add those as dependencies

Reply:
kyuubi and spark currently default to java 8 for compilation, spark uses java 19 in brew but kyuubi currently guarantees up to java 11 in runtime. I can not pass the `brew audit` verification.

Reply:
Narrowed the scope of spark dep to test to fix this. Spark's one of those SQL engines that kyuubi supports. Users are free to use kyuubi with Apache Flink, Hive, Doris; Trino, etc.